website/www/site/layouts/partials/section-menu/community.html

```diff
@@ -0,0 +1,31 @@
+<!--
```

Comment:
Do you think we should change the comment syntax here to `{{/** /}}`? Just to be sure it won't break any. 😛

Reply:
Unfortunately that syntax doesn't seem to work .md files

![image](https://user-images.githubusercontent.com/15073128/78906784-1716e380-7a80-11ea-89b4-92de0e4d6163.png)